### ISSUE TITLE
Demo content typo?

### ISSUE THREAD

**Issue by ClashTheBunny** [DanielDe/org-web#47]
![Screenshot_20190719-133457](https://user-images.githubusercontent.com/248361/61557590-3ec5a780-aa2a-11e9-8d5e-38e874f3b169.png)
It seems like the `/` is causing unintentional italics.  I don't know org formatting rules, but maybe it needs to be escaped?  If not, maybe the formatting engine is off?

**Comment by DanielDe** [comment]
Yeah, this is a bug I'm aware of that's been tricky to fix. I've got a long, ugly regex handling the parsing of a lot of the org formatting rules and it fails in cases like this. I've got plans to refactor that and fix the bugs.
